tests/test_dirhash.py

```diff
@@ -267,7 +267,7 @@ def test_cyclic_link(self):
         with pytest.raises(SymlinkRecursionError):
             filepaths = included_paths(self.path_to('root'))
 
-    def test_ignore_hidden_files(self):
+    def test_ignore_hidden(self):
```

Comment:
1. test (original) - i changed the name and the expected result for ignore codeblock, skipping '.d2/f1'